boards/arm/cyclonev_socdk/cyclonev_socdk.dts

```diff
@@ -0,0 +1,88 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2021, Intel Corporation
+ * Description:
+ * Main Device Tree source of the Cyclone V SoC DevKit
+ * this file is based on the GSRD DTS for Linux
+ */
+
+#include "intel_socfpga_std/socfpga_cyclone5.dtsi"
+#include "dt-bindings/gpio/gpio.h"
+
+/ {
+	model = "Altera SOCFPGA Cyclone V SoC Development Kit";
+	compatible = "altr,socfpga-cyclone5-socdk", "altr,socfpga-cyclone5", "altr,socfpga";
+
+	ddr0: memory@0 {
+		name = "memory";
+		device_type = "memory";
+		reg = <0x0 0x40000000>; /* 1GB */
+	};
+
+	aliases {
+	/* Giving different names to the LEDs connected to the HPS side of the chip */
+		led0 = &hps0;
+		led1 = &hps1;
+		led2 = &hps2;
+		led3 = &hps3;
+		eeprom-0 = &eeprom;
+	};
+
+	leds {
+		 compatible = "gpio-leds";
+		 hps3: hps_led_3 {
```

Comment:
nit: Space after tab here and below for the leds section

Reply:
this is fixed